### PR TITLE
Feature/ap5-101

### PR DESCRIPTION
**Pull request by jeremy-roussel**

### CODE REVIEW

**Review by BarretteBanner — APPROVED** [review]

File: rigex-front-end/src/constants/typeDefs.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ export const typeDefs = gql`
     getAllAboutText: [AboutText]
     getAllServices: [Services]
     getAllRigs: [Rigs]
+    getAallTeamMembers:  [TeamMembers]!
```
Comment (popesicle) [inline rigex-front-end/src/constants/typeDefs.js]: Typo maybe? getAallTeamMembers don't seem right...

Comment (jeremy-roussel) [reply rigex-front-end/src/constants/typeDefs.js]: My keyboard has been dying and doing weird things, typo is the correct answer

**Review by popesicle — CHANGES_REQUESTED** [review]
One comment. But it's a typo. Stop having such good PR's Jeremy. I'VE GOT NO FEEDBACK BECAUSE IT'S SO GOOD.

**Review by popesicle — APPROVED** [review]
LGTM

**Pull request merged**